x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/helpers.ts

```diff
@@ -22,7 +22,13 @@ interface StreamGraphParams {
   apmTracer: APMTracer;
   assistantGraph: DefaultAssistantGraph;
   bedrockChatEnabled: boolean;
-  inputs: { input: string };
+  inputs: {
```

Comment:
Consider extracting this type to a `GraphInputs` interface since it is repeated multiple times